### PR TITLE
Fix nits in markdown links

### PR DESCRIPTION
**Pull request by tengqm**
This PR fixes a few "bad links" identified by the `scripts/linkchecker.py` script.

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
Thanks

/approve

I see there are some whitespace and wrapping changes here too.

File: content/en/docs/concepts/windows/intro.md [hunk 1]
```diff
@@ -371,7 +371,7 @@ troubleshooting ideas prior to creating a ticket.
 
 The kubeadm tool helps you to deploy a Kubernetes cluster, providing the control
 plane to manage the cluster it, and nodes to run your workloads.
-[Adding Windows nodes](/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/)
+[Adding Windows nodes](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/)
 explains how to deploy Windows nodes to your cluster using kubeadm.
```
Comment (marosset) [inline content/en/docs/concepts/windows/intro.md:375]: It looks like the /docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/ was dropped between v1.24 and v1.25.
The updated link (/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/) does not explain how to deploy Windows nodes to the cluster link this paragraph says so we should probably just drop this for now.

There is a WIP doc in kubernetes-sigs/sig-windows-tools for this that is being reviewed https://github.com/kubernetes-sigs/sig-windows-tools/pull/239/files

Comment (tengqm) [reply content/en/docs/concepts/windows/intro.md:375]: Yes. That page was dropped and now redirected. See: https://github.com/kubernetes/website/blob/main/static/_redirects#L575

Redirections like this pose a challenge for downstream localization teams. If they use the original path, they need to add a new redirect entry per language in the `_redirects` file. We'd better avoid using redirected links in the text.

Comment (jsturtevant) [reply content/en/docs/concepts/windows/intro.md:375]: agree with @marosset the redirect doesn't really do the following "explains how to deploy Windows nodes to your cluster using kubeadm."  The steps are essentially the same but things like kubeproxy and cni's require addtional steps on windows.

Comment (marosset) [reply content/en/docs/concepts/windows/intro.md:375]: Let's just remove this entire paragraph.

Comment (tengqm) [reply content/en/docs/concepts/windows/intro.md:375]: Will do.

Comment (marosset) [reply content/en/docs/concepts/windows/intro.md]: Thanks!

File: content/en/docs/concepts/windows/intro.md [hunk 2]
```diff
@@ -371,8 +371,6 @@ troubleshooting ideas prior to creating a ticket.
 
 The kubeadm tool helps you to deploy a Kubernetes cluster, providing the control
 plane to manage the cluster it, and nodes to run your workloads.
-[Adding Windows nodes](/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/)
```
Comment (divya-mohan0209) [inline content/en/docs/concepts/windows/intro.md:374]: Did we intend to delete this, @tengqm?

**Review by divya-mohan0209 — COMMENTED** [review]
Just a single clarification sought, otherwise LGTM!

File: content/en/docs/concepts/windows/intro.md (hunk 2 above)
Comment (sftim) [reply content/en/docs/concepts/windows/intro.md:374]: https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/ is no longer there (we redirect to the most similar page that does exist).

Comment (tengqm) [reply content/en/docs/concepts/windows/intro.md:374]: > Did we intend to delete this, @tengqm?

Yes. It was suggested by the Windows contents "owner".

Comment (divya-mohan0209) [reply content/en/docs/concepts/windows/intro.md:374]: Thank you for the clarification, folks! Much appreciated :)

**Comment by divya-mohan0209** [comment]
@tengqm if you can resolve the conflicts, I think we're good to go ahead with the merge.

**Comment by windsonsea** [comment]
/lgtm
thanks.

**Pull request merged**